packages/analytics/dashboard-renderer/package.json

```diff
@@ -66,13 +68,14 @@
   },
   "peerDependencies": {
     "@kong-ui-public/analytics-chart": "workspace:^",
+    "@kong-ui-public/analytics-utilities": "workspace:^",
     "@kong-ui-public/i18n": "workspace:^",
     "@kong/kongponents": "^8.126.2",
+    "swrv": "^1.0.4",
```

Comment:
docs: I think these peerDependencies should be documented in the README with the others?

Reply:
Done.